storage/storage-service/src/lib.rs

```diff
@@ -304,6 +305,50 @@ impl Storage for StorageService {
         let resp = self.get_epoch_change_ledger_infos_inner(req);
         provide_grpc_response(resp, ctx, sink);
     }
+
+    fn backup_account_state(
+        &mut self,
+        ctx: grpcio::RpcContext,
+        req: BackupAccountStateRequest,
+        sink: grpcio::ServerStreamingSink<BackupAccountStateResponse>,
+    ) {
+        debug!("[GRPC] Storage::backup_account_state");
+        let _timer = SVC_COUNTERS.req(&ctx);
+        let mut success = true;
+        let iter = self.db.get_account_iter(req.version);
+        match iter {
+            Ok(iter) => {
+                let f = sink
+                    .send_all(futures01::stream::iter_result(iter.map(|res| match res {
+                        Ok((hash, blob)) => Ok((
+                            BackupAccountStateResponse {
+                                account_key: hash.to_vec(),
+                                account_state_blob: Some(blob.into()),
+                            },
```

Comment:
Just use the rust->proto conversion?

Reply:
Doesn't work - the item type is a tuple - `type Item = Result<(HashValue, AccountStateBlob)>;`